src/test/java/org/prebid/server/bidder/pubmatic/PubmaticBidderTest.java

```diff
@@ -932,6 +942,13 @@ private static ExtRequest givenUpdatedBidRequestExt(Integer wrapperProfile, Inte
         return jacksonMapper.fillExtension(ExtRequest.empty(), wrapperNode);
     }
 
+    @NotNull
+    private ExtRequest givenExtRequest() {
```

Comment:
Better do it inside test case as we do not (and probably will not need it) anywhere else.